framework/mesh/mesh_continuum/mesh_continuum.cc

```diff
@@ -931,103 +931,69 @@ bool
 MeshContinuum::CheckPointInsideCell(const Cell& cell, const Vector3& point) const
 {
   const auto& grid_ref = *this;
-  typedef Vector3 Vec3;
-  auto InsideTet = [](const Vec3& point, const Vec3& v0, const Vec3& v1, const Vec3& v2)
+
+  auto InsideTet = [](const Vector3& point, const Vector3& v0, const Vector3& v1, const Vector3& v2)
   {
     const auto& v01 = v1 - v0;
     const auto& v02 = v2 - v0;
-
     const auto n = v01.Cross(v02).Normalized();
     const auto c = (v0 + v1 + v2) / 3.0;
-
     const auto pc = point - c;
 
     if (pc.Dot(n) > 0.0)
       return true;
-    else
-      return false;
+
+    return false;
   };
```

Comment:
Just something to consider: This could be moved to be located with the check for polyhedra.

Reply:
Fixed.